chore: upgrade to capacitor v7

JanStevens opened the pull request:
Output of running npx @capacitor/plugin-migration-v6-to-v7@latest

Fixes #36 

Since capacitor v7 is a small bump for min android version this seemed trivial to add.

JanStevens commented:
@AlwaysLoveme Any feedback on getting this merged? Currently this plugin prevents us from upgrading to capacitor v7 🙏

AlwaysLoveme commented:
> @AlwaysLoveme Any feedback on getting this merged? Currently this plugin prevents us from upgrading to capacitor v7 🙏

are you tested?

gurucomkz commented:
@JanStevens May I suggest to remove the cosmetic changes and leave only the changes that are essential to this issue? See, the count of the changed files is currently 22 which makes it rather hard to find focus on the issue. Thank you.

JanStevens commented:
@gurucomkz I've updated the PR to only include the changes done by the upgrade script and some readme / version bump changes for convenience.

@AlwaysLoveme I've used the example app and added some code to test the functionality of the plugin (not included in this PR but let me know if I should commit it in a different PR).

### Android

| Pixel 4 (API 30) | Pixel 9 (API 35) | Output |
|------------------|------------------|--------|
|![pixel-4-api-30](https://github.com/user-attachments/assets/bd4f24dc-436a-4942-8df8-9c8ea2b7cb26)|![pixel-9-api-35](https://github.com/user-attachments/assets/01a85ff0-73ad-45a6-9181-6c5d6bf4d294)| ![android-plugin-output](https://github.com/user-attachments/assets/f2fc014a-5a78-4cf7-9030-2070d89fd757) |

### IOS

| iPhone 14 (iOS 16.4) | iPhone 16 (iOS 18.2) | Output |
|----------------------|----------------------|--------|
|![iphone-14-ios-16](https://github.com/user-attachments/assets/1e835760-9c3e-427a-ae5e-efbe5622e40e)|![iphone-16-ios-18](https://github.com/user-attachments/assets/1599410c-974c-4d5e-bb13-ba6181f55644)|![ios-plugin-output](https://github.com/user-attachments/assets/297e05bc-4b9a-4ffb-b8cd-a4414b1d2fad)|

If you want me to test with different specific devices let me know.

gurucomkz commented:
@JanStevens Thank you very much for that!

@AlwaysLoveme  I've tested this with Android and it looked very well. Yet to test with iOS, though.

AlwaysLoveme commented:
> @JanStevens Thank you very much for that!
> 
> @AlwaysLoveme I've tested this with Android and it looked very well. Yet to test with iOS, though.

already published
